packages/hint-ie-flexbox-compat/src/_locales/en/messages.json

```
@@ -0,0 +1,14 @@
+{
+    "description": {
+        "description": "Metadata description",
+        "message": "Informs users that Internet Explorer suffers from Flexbox compatibility bugs."
+    },
+    "name": {
+        "description": "Metadata name",
+        "message": "IE Flexbox Compatibility"
+    },
+    "usingFlexWithIE": {
+        "description": "String warning users that flex is used and IE is a target browser",
+        "message": "The Internet Explorer brower versions 11 and older suffer from a number of CSS Flexbox compatibility bugs. The layout on this page may be different between Internet Explorer and other, more recent, browsers. The size or alignment of elements may be incorrect and content may overflow.\nSupport for older versions of Internet Explorer ended and Microsoft no longer provides security updates or technical support for these versions."
```

Let's keep the message a bit more concise and leave the details for the README.

```suggestion
        "message": "This page uses CSS Flexbox which may render differently in Internet Explorer than more recent browsers. Double-check this page in IE then see the documentation for workarounds if needed."
```